gapir/cc/replay_connection.h

```diff
@@ -225,9 +226,9 @@ class ReplayConnection {
   // Sends post data. Returns true if succeeded, otherwise returns false.
   virtual bool sendPostData(std::unique_ptr<Posts> posts);
   // Sends notification. Returns true if succeeded, otherwise returns false.
-  virtual bool sendNotification(uint64_t id, uint32_t api_index, uint64_t label,
-                                const std::string& msg, const void* data,
-                                uint32_t data_size);
+  virtual bool sendNotification(uint64_t id, int severity, uint32_t api_index,
```

Comment:
Can we make severity unsigned? Is there any reason it can ever be negative?

Reply:
Changed to uint32_t in c0e75da